crates/triangulation/src/path_triangulation.rs

```diff
@@ -220,3 +220,28 @@ pub fn triangulate_path_edge(
     result.fix_triangle_orientation();
     result
 }
+
+pub fn triangulate_paths_edge(
```

Comment:
Function name is a bit unclear. What does it mean to triangulate a path's edge? Is this "Convert paths represented by vectors to a Path Triangulation form."?

Reply:
Added docstring. Will think about better name.

Reply:
I think the name is fine with the docstring. 👍🏼